modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -308,7 +308,7 @@ export async function getListTypeDefinitionListTypeEntries(listTypeDefinitionId:
 	);
 }
 
-export async function getPickLists() {
+export async function getListTypeDefinitions() {
```

Comment:
@igor-franca check sorting here too.

Reply:
Checked! It has already been done in the sort commit